lib/convection/model/template/resource/aws_ec2_vpn_connection.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        ##
+        # AWS::EC2::VPNConnection
+        ##
+        class EC2VPNConnection < Resource
+          include Model::Mixin::Taggable
+
+          type 'AWS::EC2::VPNConnection'
+          property :conn_type, 'Type'
```

Comment:
Why use `:conn_type`? Let's use `connection_type` since `type` is taken. It's more clear.